tools/pylint-extensions/pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -875,8 +875,8 @@ class ClientListMethodsUseCorePaging(BaseChecker):
 
     def __init__(self, linter=None):
         super(ClientListMethodsUseCorePaging, self).__init__(linter)
-
-    def visit_functiondef(self, node):
+    
+    def visit_return(self, node):
```

Comment:
Assuming this change is to make the search more direct?

Reply:
yes!